elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/annotation/Cardinality.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates whether a dimension has small, medium, or large cardinality.
+ * <p>
+ * See {@link DimensionSize}.
+ * <p>
+ * Example: {@literal @}Cardinality(size = {@link DimensionSize#MEDIUM})
+ */
+@Documented
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Cardinality {
+
+    /**
+     * Returns the size category of a dimension.
+     * <p>
+     * The size category must be from one of the values of type {@link DimensionSize}.
+     *
+     * @return dimension size
+     */
+    DimensionSize size();
```

Comment:
How about a default value.  The default should be large.

Reply:
Sure